### ISSUE TITLE
Server working but not showing up in Community servers

### ISSUE THREAD

**Issue by s1501** [thijsvanloef/palworld-server-docker#24]
**Describe the bug**
Set the community server env variable to true, the server doesn't show up in the community servers list, the port forwarding is correct, and direct connecting with ip:ports works if the password is removed, but the server doesn't show up in either the server browser or the Recent server list, I know there are problems right now with the server search, but it doesnt show up even looking here: https://www.battlemetrics.com/servers/palworld?sort=score

Anyone knows whats wrong, I'm self hosting it on my home server with a dynamic ip, sadly the game doesn't let you connect using an url, so I'd like to make this work so I don't have to tell all my friend my new ip every time it refreshes.

**Comment by MoryCorp** [comment]
Hey !  

The server list is having a lot of issues right now due to the high amount of servers being created.  
You may need to wait or force the refresh with the pagination "show 200 more" to see your server !

**Comment by rezeropoint** [comment]
> Hey !
> 
> The server list is having a lot of issues right now due to the high amount of servers being created. You may need to wait or force the refresh with the pagination "show 200 more" to see your server !

Unable to find even when there are less than 200 servers

**Comment by aptlt** [comment]
Hey, same here. I tried to disable the community option and the password option and was able to connect directly from IP_ADDRESS:PORT. So pretty sure the problem isn't an connection issue...

**Comment by martinmiglio** [comment]
The community servers list takes time to update. I deployed a server today using this image and it took almost an hour for it to show up. 

This is not an issue with this docker image or the server executable it is based on.

**Comment by aptlt** [comment]
Yep, seems great having the information though, avoiding us losing time :) Maybe we can write it into the Knowledges issues ?

**Comment by martinmiglio** [comment]
[This gist](https://gist.github.com/Toakan/3c78a577c21a21fcc5fa917f3021d70e) has info on this topic (+ lots more).
